R/readFiles.R

```diff
@@ -215,6 +215,9 @@ read_IndexFile_BIOBJ_COCO <- function(fname) {
 #' path <- system.file("extdata", "ONE_PLUS_LAMDA_EA", package="IOHanalyzer")
 #' check_format(path)
 check_format <- function(path) {
+  if (sub('[^\\.]*\\.', '', basename(path), perl = T) == "csv")
```

Comment:
checking .csv suffix should be fine for now. But, I think it is better to check the header (structure) of those csv, to make sure.

Reply:
Yes, that is true. I will add it to the trello.